ckanext/push_errors/logging.py

```diff
@@ -12,6 +13,51 @@
 log = logging.getLogger(__name__)
 
 
+# Configuración de Redis
+redis_client = redis.StrictRedis(
+    host=toolkit.config.get('ckan.redis.url', 'redis_bcie'),
+    port=6379,
+    db=0
+)
```

Comment:
Using something like CKAN code to connect

```py
from redis import Redis, ConnectionPool

...

redis_url = config.get('ckan.redis.url', 'redis://localhost:6379/0')
redis_pool = ConnectionPool.from_url(redis_url)
cache = Redis(connection_pool=redis_pool)from redis
```

Then we can use `cache.incr("push_errors:minute:xxx")` and `cache.get("push_errors:minute:xxx")` to create/increment and to get the value